.github/workflows/docs-release.yml

```
@@ -0,0 +1,70 @@
+name: Rebuild docs with the latest release
+
+on:
+  release:
```

```suggestion
  release:
    types: [released]
```

As we want this to run on stable releases only, not necessarily all release related actions.

---

Corresponding GitHub doc pages for reference:
- [`release` event](https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#release)
- [`released` webhook](https://docs.github.com/en/webhooks/webhook-events-and-payloads?actionType=released#release) (i.e. the type)